### ISSUE TITLE
Make more faces customizable

### ISSUE THREAD

**Issue by sheijk** [oantolin/embark#104]
Using embark with my theme the prompter menu looks like this
![embark-color-conflict](https://user-images.githubusercontent.com/44696/104140698-f551e080-53b2-11eb-91fa-5b12b6d13be4.png)
which makes it hard to read. The purple color is `minibuffer-prompt`. After typing more than one character the purple background disappears, but the propertize-d text still hides the current selection highlight.

I'm using selectrum version 20210110.2134. If I'm switching to ivy the problem disappears.

In line 551 (as of 20210110.1409 in Melpa) in embark-completing-read-prompter the used faces are hard-coded to 'success and 'shadow. Could these be made customizable?

**Comment by oantolin** [comment]
Sure, I shouldn't have hard coded them in the first place. I'll make new faces that just inherit from those as a default.

**Comment by sheijk** [comment]
Great, that makes it usable for me with a foreground and distant-foreground color.

I think there are still some display issues I don't see with ivy but I'm not sure whether they're embark or selectrum issues:
![embark-selectrum-issue](https://user-images.githubusercontent.com/44696/104231238-f33d5f80-544e-11eb-822a-291feb5cd356.png)
Left: selectrum with no input, middle: selectrum with one char, right: ivy with no input

minibuffer-prompt color seems to bleed into the menu and embark's Act gets put after the minibuffer prompt in selectrum but not ivy. I've set embark-prompter to 'embark-completing-read-prompter. I also need to type one character to see ivy/selectrum - is that intended?

**Comment by sheijk** [comment]
The issue with the completion system only being shown after one key press also does not happen when I'm operating on selectrum/ivy selections, only if invoked from a (mini)buffer

**Comment by oantolin** [comment]
I have no idea what's going on with the colors. Or with the `Act` indicator in Ivy. I'll try to look into it, but I'm not an expert on either Selectrum or Ivy.

> The issue with the completion system only being shown after one key press also does not happen when I'm operating on selectrum/ivy selections, only if invoked from a (mini)buffer

This is a known problem: #97. I know how to fix it in an ugly way, and am waiting for some inspiration to fix it in a nice way.

Since you mention Ivy, can I ask you a question about it? Are you using the functions from #6? If so, do the file actions work when you navigate to other directories? I suspect those functions might need to return full paths instead of relative paths or something like that.

**Comment by sheijk** [comment]
I see the same glitch with the ACT prompt sliding to the end with ivy and selectrum. I tried the first code snippet from [here](https://github.com/oantolin/embark/issues/6#issuecomment-756338938) but didn't see any change using `C-h f` (not sure what to look for?). But I'm only mostly using ivy to see if some error happens with selectrum only.

**Comment by sheijk** [comment]
The issue is gone with this simple prompter:
```elisp
(defun shk-embark-transient-prompter (keymap)
  "Select an embark action using a transient menu"
  (let ((commands (mapcar (lambda (item) (cdr item))
                          (cdr (keymap-canonicalize keymap)))))
    (intern-soft (completing-read "command: " commands))))
```

**Comment by sheijk** [comment]
Can confirm that the fix for raxod502/selectrum#235 also fixes the display issues for me
